presto-product-tests/src/main/java/io/prestosql/tests/kafka/TestKafkaAvroWritesSmokeTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.kafka;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.Requirement;
+import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.Requires;
+import io.prestosql.tempto.configuration.Configuration;
+import io.prestosql.tempto.fulfillment.table.kafka.KafkaTableDefinition;
+import io.prestosql.tempto.fulfillment.table.kafka.ListKafkaDataSource;
+import org.testng.annotations.Test;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.fulfillment.table.TableRequirements.immutableTable;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.KAFKA;
+import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static java.lang.String.format;
+import static org.testng.Assert.assertTrue;
+
+public class TestKafkaAvroWritesSmokeTest
+        extends ProductTest
+{
+    private static final String KAFKA_CATALOG = "kafka";
+
+    private static final String ALL_DATATYPES_AVRO_TABLE_NAME = "product_tests.write_all_datatypes_avro";
+    private static final String ALL_DATATYPES_AVRO_TOPIC_NAME = "write_all_datatypes_avro";
+
+    private static final String ALL_NULL_AVRO_TABLE_NAME = "product_tests.write_all_null_avro";
```

Comment:
why do we need separate table for writing nulls. Can't we write to `product_tests.write_all_datatypes_avro`?

Reply:
All null values is encoded as an empty message in Kafka. All nulls looks different from a null column within Kafka.

I'll add a NULL value test in `write_all_datatypes_*` tests as I've missed those.

Reply:
Oh, I got what you meant. We won't need this ALL_NULL table once we have an all null test case in the write_all_datatypes. 👍